pytorch_to_returnn/torch/nn/modules/operator.py

```diff
@@ -285,6 +285,24 @@ def create_returnn_layer_dict(self, input: Tensor) -> Dict[str, Any]:
       "position": self.pos}
 
 
+class GatherTensor(Module):
+  """
+  Basically x[pos] but in specific dim (axis) where pos is a tensor.
+  """
+  is_original_torch_module = False
+
+  def __init__(self, dim: int):
+    super(GatherTensor, self).__init__()
+    self.dim = dim
+
+  def create_returnn_layer_dict(self, input: Tensor, pos: Tensor) -> Dict[str, Any]:
+    pos = self._get_input_layer_name(pos)
+    return {
+      "class": "gather", "from": self._get_input_layer_name(input),
+      "axis": self._get_input_axis_to_returnn(input, axis=self.dim),
+      "position": pos}
```

Comment:
I think this looks a bit more consistent:
```suggestion
    return {
      "class": "gather", "from": self._get_input_layer_name(input),
      "axis": self._get_input_axis_to_returnn(input, axis=self.dim),
      "position": self._get_input_layer_name(pos)}
```